json-schema/schema.json

```diff
@@ -295,6 +295,10 @@
           "description": "Focal length in mm",
           "type": "number"
         },
+        "field_of_view": {
+          "description": "Field of view (horizontal extent observable) in degrees",
+          "type": "number"
```

Comment:
I assume here (and in focal_length) an `"minimumExclusive": 0` would make sense? Probably for most other numerical values in this extension, too?

Reply:
Hi, thanks for this feedback. I added the constraint on fields I'm sure this makes sense. It may be useful on some others, but as I'm no photo expert, I prefer to let someone else add it where necessary :wink: